source/installation_guide/upgrading/from-2.3-to-3.0.rst

```diff
@@ -93,6 +82,27 @@ Removed features and their replacements
 +------------------------------------------------------------+------------------------------------------------------------------------------------------+
 | checkpassword auth database                                | Use :ref:`authentication-lua_based_authentication` instead.                              |
 +------------------------------------------------------------+------------------------------------------------------------------------------------------+
+| IPC process                                                | Has been merged to anvil.                                                                |
++------------------------------------------------------------+------------------------------------------------------------------------------------------+
+| ``login_access_sockets``                                   | Use :ref:`authentication-lua_based_authentication` instead.                              |
++------------------------------------------------------------+------------------------------------------------------------------------------------------+
+
+
+Changed default settings
+========================
+
++------------------------------------------------+----------------------------+------------------+-------------------------------------------------------------+
+| Key                                            | Old                        | New              | Notes                                                       |
++================================================+============================+==================+=============================================================+
+| service/anvil/chroot                           | empty                      | <no value>       | Anvil is no longer chrooted                                 |
++------------------------------------------------+----------------------------+------------------+-------------------------------------------------------------+
+| service/anvil/user                             | $default_internal_user     | <no value>       | Anvil runs as root                                          |
++------------------------------------------------+----------------------------+------------------+-------------------------------------------------------------+
+| service/auth-worker/process_limit              | 1                          | 30               | Behaviour of process limit has changed for auth-worker,  it |
+|                                                |                            |                  | now behaves as it was supposed to.                          |
++------------------------------------------------+----------------------------+------------------+-------------------------------------------------------------+
+| submission_add_received_header                 | <not present>              | yes              | See :dovecot_core:ref:`submission_add_received_header`      |
```

Comment:
Looks like a new setting, not a changed default? Especially because the default is the same as the old behavior was anyway.